CHANGES

```diff
@@ -4,6 +4,16 @@ For a complete changelog, see:
 * https://github.com/yaml/pyyaml/commits/
 * https://bitbucket.org/xi/pyyaml/commits/
 
+6.0b1 (2021-09-29)
+
+* https://github.com/yaml/pyyaml/pull/497 -- fix float resolver to ignore `.` and `._`
+* https://github.com/yaml/pyyaml/pull/550 -- drop Python 2.7
+* https://github.com/yaml/pyyaml/pull/556 -- fix representation of Enum subclasses
+* https://github.com/yaml/pyyaml/pull/557 -- fix libyaml extension compiler warnings
+* https://github.com/yaml/pyyaml/pull/560 -- fix ResourceWarning on leaked file descriptors
+* https://github.com/yaml/pyyaml/pull/561 -- always require `Loader` arg to `yaml.load()`
+* https://github.com/yaml/pyyaml/pull/564 -- remove remaining direct distutils usage
+
```

Comment:
I would add those 3 to be complete:
https://github.com/yaml/pyyaml/pull/327 Change README format to Markdown
https://github.com/yaml/pyyaml/pull/483 Add a test for YAML 1.1 types
https://github.com/yaml/pyyaml/pull/553 Fix spelling of “hexadecimal”

Reply:
will do- thanks!